Documentation/Meta/Index.rst

```diff
@@ -0,0 +1,26 @@
+.. include:: ../Includes.txt
```

Comment:
I guess we don't need this file

Reply:
was in the template, so would keep it or ask daniel from the docs team.